docs/docs/integrations/llms/runhouse.ipynb

```diff
@@ -51,16 +43,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": null,
    "id": "06d6866e",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
     "# For an on-demand A100 with GCP, Azure, or Lambda\n",
-    "gpu = rh.cluster(name=\"rh-a10x\", instance_type=\"A100:1\", use_spot=False)\n",
-    "\n",
+    "gpu = rh.cluster(name=\"langchain-rh-a10x\", instance_type=\"g5.4xlarge\", provider=\"aws\")\n",
```

Comment:
maybe better to keep without specifying a provider? and use the default "cheapest" behavior